### ISSUE TITLE
Add ability to have multiple raise sizes in Range Builder

### ISSUE THREAD

**Issue by danielc2501** [lukebhan/memorizePreFlop#5]
It's common in vs 3bet situations for example, for the GTO response to include some 4bets to a small size, and some all-ins. It would be great if this was made possible in the chart builder.

**Comment by lukebhan** [comment]
Hi Daneil,
Thank you very much for your suggestions - I think they are great and would happily look if you implemented them for the project and submitted a pull request. However, I am not actively adding to this project with new features and actually if you wanted most of these features for a price, there is https://freebetrange.com/en that offers most of what you want for a subscription price. With that said, I will be closing the issues you opened as they are more so wanted features and I don't plan on actively adding to this project. Thanks for checking out the project!

**Comment by danielc2501** [comment]
Hi Luke,

Thanks very much for the update.

It's possible that I do end up submitting some pull requests as a side
project for myself at some point.

I wasn't aware of freebetrange, so that looks like a great option for what
I'm looking for at the moment, so thanks for sharing that with me.

I was impressed with the work you put in to this project. Well done. I hope
you're still enjoying poker :)

**Comment by lukebhan** [comment]
Definitely. Ill keep this one open then and feel free to ping me here or recreate any if you make a pull request. And yes freebetrange is great - and they have a nice free trial that I used. Thanks for checking out the project!
